truss/templates/server/common/errors.py

```diff
@@ -113,60 +126,67 @@ async def exception_handler(
     NotImplementedError,
     InputParsingError,
     UserCodeError,
+    ModelDefinitionError,
     fastapi.HTTPException,
 }
 
 
-def _intercept_user_exception(exc: Exception, logger: logging.Logger) -> NoReturn:
+def _filter_traceback() -> (
+    Union[
+        Tuple[Type[BaseException], BaseException, TracebackType],
+        Tuple[None, None, None],
+    ]
+):
+    exc_type, exc_value, tb = sys.exc_info()
+    if tb is None:
+        return exc_type, exc_value, tb  # type: ignore[return-value]
+    extracted_tb = traceback.extract_tb(tb)
+    # Find the first frame with 'model.py' and slice from that frame onward.
+    for idx, frame in enumerate(extracted_tb):
+        if frame.filename.endswith("model.py"):
+            # Slice from the 'model.py' frame onward
```

Comment:
awesome

Reply:
I am incredibly excited for this feature. One important note: the model file is not _necessarily_ called "model.py". There is a config option for this: https://github.com/basetenlabs/truss/blob/a121e69bdb4a5519a19a78aa2bc16c6672c82ec1/truss/truss_config.py#L487, let's make sure that we're using that